plugins/scale.lua

```diff
@@ -96,14 +96,14 @@ end
 
 command.add(nil, {
   ["scale:reset"   ] = function() set_scale(default)             end,
-  ["scale:decrease"] = function() set_scale(current_scale * 0.9) end,
+  ["scale:decrease"] = function() set_scale(current_scale / 1.1) end,
```

Comment:
why exactly? it is preferable to use multiplications than divisions.

Reply:
> why exactly?
Because it works right. Just take it and count it.
> it is preferable to use multiplications than divisions.
Why is it preferable?

Reply:
multiplications are faster in most languages. The difference is not much in Lua. It's still a good habbit.

Reply:
Stop trolling me. Better do something useful.

Reply:
I'm sorry you are taking it that way, is not my intention to troll anybody.

Reply:
The performance argument isn't even funny. If we were talking about billions of calculations with budget of 16 milliseconds, then your words would make sense. But this operation is performed once after pressing the button. Even a million divisions will work quickly and the user won't notice the difference.
I don't know what else to add. This should be obvious, especially in a world where there is a phenomenon of electron applications.

```
x * 1.1 / 1.1 == x
x * 1.1 * 0.9 ~= x
```
How can multiplication be preferable if it gives a different result?

Reply:
thank you for your explanation, I'm not going to argue about this.

Reply:
The scaling is kind of weird. Together with the scalestatus.lua plugin, when scaling it shows 101%, 78% when going back and forth between them.
I changed:
```lua
command.add(nil, {
  ["scale:reset"   ] = function() set_scale(default)             end,
  ["scale:decrease"] = function() set_scale(current_scale * 0.9) end,
  ["scale:increase"] = function() set_scale(current_scale * 1.1) end,
})
```
to:
```lua
command.add(nil, {
  ["scale:reset"   ] = function() set_scale(default)             end,
  ["scale:decrease"] = function() set_scale(current_scale - 0.1) end,
  ["scale:increase"] = function() set_scale(current_scale + 0.1) end,
})
```
The number is cleaner (70%, 80%, 90%, 100%, ...) and if we're talking performance then this is also better.

Reply:
I like this approach. Would prefer if the increment were a setting so it can be changed depending on hardware.